compiler/rustc/src/main.rs

```diff
@@ -1,3 +1,11 @@
+// FIXME(guswynn): depending on perf results, document this thoroughly
```

Comment:
Looks like this FIXME (and the related one in rustdoc) could use fleshing out.

Reply:
I would prefer rustdoc just links to the rustc docs, otherwise people will forgot to update them.